integration_tests/saiutil/saiutil.go

```diff
@@ -30,6 +45,19 @@ func dataplaneConn(t testing.TB, dut *ondatra.DUTDevice) *grpc.ClientConn {
 	return conn
 }
 
+type configer struct {
+	cfg   func(t testing.TB, s *Suite, dut *ondatra.DUTDevice)
```

Comment:
Should these funcs return error?

Reply:
these are test helpers so the fail the test instead of returning err